sympy/stats/symbolic_probability.py

```diff
@@ -59,6 +59,7 @@ class Probability(Expr):
     >>> prob.evaluate_integral()
     sqrt(2)*(-sqrt(2)*sqrt(pi)*erf(sqrt(2)/2) + sqrt(2)*sqrt(pi))/(4*sqrt(pi))
     """
+    is_commutative = True
     def __new__(cls, prob, condition=None, **kwargs):
```

Comment:
There should be a blank line above and below the `is_commutative` line.

Reply:
Sure !